src/utils/src/Thread.c

```diff
@@ -51,7 +51,7 @@ PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID
     pWrapper->storedArgs = args;
     pWrapper->storedStartRoutine = start;
 
-    threadHandle = CreateThread(NULL, 0, startWrapperRoutine, pWrapper, 0, NULL);
+    threadHandle = CreateThread(NULL, stackSize, startWrapperRoutine, pWrapper, 0, NULL);
```

Comment:
Do we know what the default min stack size for Windows is? Can we have a unit test for all Linux and Ubuntu to test out the negative return codes too (specifically looking for setting a really low value to confirm the failure status code.

Reply:
I tried looking for that but I couldn't find a source

Reply:
There doesn't appear to be a minimum stack size for windows that I can find in any documentation

Reply:
the failure status code of linux is 22, invalid parameter. For Windows it will just crash from stack overflow if you provide an incredibly low value.